source/common/upstream/subset_lb.cc

```diff
@@ -158,10 +162,8 @@ void SubsetLoadBalancer::initSelectorFallbackSubset(
                              LbSubsetSelector::ANY_ENDPOINT &&
       selector_fallback_subset_any_ == nullptr) {
     ENVOY_LOG(debug, "subset lb: creating any-endpoint fallback load balancer for selector");
-    HostPredicate predicate = [](const Host&) -> bool { return true; };
-    selector_fallback_subset_any_ = std::make_shared<LbSubsetEntry>();
-    selector_fallback_subset_any_->priority_subset_.reset(
-        new PrioritySubsetImpl(*this, predicate, locality_weight_aware_, scale_locality_weight_));
+    initSubsetAnyOnce();
+    selector_fallback_subset_any_ = subset_any_;
```

Comment:
Nice cleanup! is there any reason now to keep `selector_fallback_subset_any_` field?

Reply:
I'm not sure if we'll add a stats metric like 'lb_subsets_fallback_panic_'. So we decided to keep this field separate from the panic

Reply:
@zyfjeff agree on panic case, i meant cannot we reuse `subset_any_` straight away instead of `selector_fallback_subset_any_` for subset selectors?

Reply:
think that's ok. Let me get rid of this field.